drivers/volume/portworx/portworx.go

```diff
@@ -718,6 +718,19 @@ func (d *portworx) CreateVolume(volName string, size uint64, haLevel int64) (str
 	return resp.VolumeId, nil
 }
 
+func (d *portworx) CreateVolumeUsingRequest(request *api.SdkVolumeCreateRequest) (string, error) {
+	volDriver := d.getVolDriver()
+	resp, err := volDriver.Create(d.getContext(), request)
+	if err != nil {
+		err = fmt.Errorf("error while creating volume because of: %v", err)
+		return "", err
+	}
+
+	d.log.Infof("successfully created Portworx volume %v ", resp.VolumeId)
```

Comment:
Nit: Can we just start with capital `S` like in other `log.Info()` to be consistent

Reply:
fixed